proc.c

```diff
@@ -439,13 +439,13 @@ scheduler(void)
         switch (pr->state)
         {
           case RUNNING:
-            pr->rutime++;
+            pr->rutime += (delta_ticks / QUANTA);
```

Comment:
i foiled at git again. could you throw this
  if(proc && proc->state == RUNNING && tf->trapno == T_IRQ0+IRQ_TIMER)
  {
    quanta_ticks++;
    if (quanta_ticks >= QUANTA)
    {
      quanta_ticks = 0;
      yield();
    }
  }

at line 105 of trap.c and add
uint quanta_ticks = 0;
at the top pls?